src/components/index.ts

```diff
@@ -1 +1 @@
-export * from './navigation';
+export { default } from './Navbar';
```

Comment:
This is where I mean you have to name your export, in this case export { default as Navbar }, this just allows you to do nesting exports from a top level file. This idea comes up in main.tsx

Reply:
Initially I was confused by this, but I see what you mean now. You have to alias exports, otherwise all the defaults in the top-level `index.ts` will clash.

Reply:
Please see https://github.com/UNLV-CS472-672/2024-S-GROUP1-Roadwatch/pull/12#discussion_r1509897794